data/channels.csv

```diff
@@ -21239,31 +21239,31 @@ NLTV55.us,NLTV 55,,,,US,US-HI,Hilo,s/US-HI,eng,general,FALSE,,,,https://naleo.tv
 Noise.ru,Noise,,,Slow TV,RU,,,c/RU,rus,,FALSE,2016-01-01,2018-01-20,,http://tvslow.com/,https://i.imgur.com/pyqpmZs.png
 NoiTV.it,NoiTV,,,,IT,,Lucca,c/IT,ita,general,FALSE,1989-01-01,,,https://www.noitv.it/live/,https://i0.wp.com/www.noitv.it/wp-content/uploads/2023/05/logo_noitv_new2017_claim_sm.png?w=500&ssl=1
-NojoumTV.jo,Nojoum TV,,,,JO,,,c/JO,ara,music,FALSE,,,,,https://i.imgur.com/6ttWNu0.jpg
+NojoumTV.ae,Nojoum TV,,,Nojoum Media,AE,,Dubai,c/AE;c/JO,ara,music,FALSE,,,,https://nojoumtv.com/,https://i.imgur.com/6ttWNu0.jpg
 NokshiTV.bd,Nokshi TV,,,,BD,,Dhaka,c/BD,ben,entertainment,FALSE,2015-08-24,,,https://nokshitv.com/,https://i.imgur.com/QWJNqS5.jpeg
 Nolife.fr,Nolife,,,Nolife S A,FR,,,c/FR,fra,entertainment,FALSE,2007-06-01,2018-04-30,,https://nolife-tv.com,https://i.imgur.com/hyDi9vT.png
-NollyAfrica.uk,Nolly Africa,,,,UK,,,c/UK,eng,,FALSE,,,,https://www.africanmoviechannel.tv/products-and-services,https://i.imgur.com/IxtGpti.png
+NollyAfrica.uk,Nolly Africa,,,,UK,,,c/UK,eng,movies,FALSE,,,,https://www.africanmoviechannel.tv/products-and-services,https://i.imgur.com/IxtGpti.png
 NollywoodCapitalTV.ng,Nollywood Capital TV,,,,NG,,Lagos,c/NG,eng,entertainment,FALSE,,,,https://nollywoodcapitaltv.com/watch/46626541,https://image.roku.com/developer_channels/prod/61530e2a25ef79aaa99eaf301a1135cb87f81fdfcd8af5be33c700cc772db2b8.png
 NollywoodTV.fr,Nollywood TV,,Les Bouquets Canal+,Thema,FR,,Paris,c/FR,fra,movies,FALSE,2012-10-05,,,https://www.nollywoodtv.fr/,https://i.imgur.com/29L2gbb.png
 NollywoodTVEpic.fr,Nollywood TV Epic,Nollywood TV Épic,Les Bouquets Canal+,Thema,FR,,Paris,c/FR,fra,entertainment,FALSE,,,,https://www.nollywoodtv.fr/,https://i.imgur.com/7krWz4D.png
 NongomaTV.za,Nongoma TV,,,,ZA,,Nongoma,c/ZA,eng;zul,culture,FALSE,,,,https://nongomatv.com/,https://i.imgur.com/j217sVx.jpg
-NONN1TV.at,NON-N1-TV,NÖN-N1-TV;N1 Niederösterreich TV,,,AT,,,c/AT,deu,general,FALSE,,,,https://www.n1tv.at/,https://i.imgur.com/HN90fAa.png
+NONN1TV.at,NON-N1-TV,NÖN-N1-TV;N1 Niederösterreich TV,,,AT,,Vienna,c/AT,deu,general,FALSE,,,,https://www.n1tv.at/,https://i.imgur.com/HN90fAa.png
 NonStopPeople.fr,Non Stop People,,,Banijay,FR,,Paris,c/FR,fra,general,FALSE,2012-11-19,2021-11-17,,http://non-stop-tv.fr/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/47/Logo_Non_Stop_People.svg/langfr-512px-Logo_Non_Stop_People.svg.png
 NoorDubai.ae,Noor Dubai,نور دبي تي في,,Dubai Media Incorporated,AE,,Dubai,c/AE,ara,religious,FALSE,,,,https://www.noordubai.com/,https://i.imgur.com/DLe7ZuM.png
 NoorTelevisao.mz,Noor Televisao,Noor Televisão,,,MZ,,Maputo,c/MZ,por,religious,FALSE,,,,,https://i.imgur.com/uUgO6L9.jpg
-NoorTV.af,Noor TV,,,,AF,,,c/AF,fas,general,FALSE,,,,https://www.noortv.af/,https://i.imgur.com/GKp7m0V.png
+NoorinTV.af,Noorin TV,تلویزیون نورین,,Haji Mohammad Arif Noori,AF,,Kabul,c/AF,fas,general,FALSE,2007-01-01,2021-12-01,,https://www.noortv.af/,https://i.imgur.com/GKp7m0V.png

```

Comment:
Are you sure "Noor TV" and "Noorin TV" aren't two different channels? 

There is no mention of any other name on Noor TV's YouTube channel: https://www.youtube.com/@NoorTV_af/videos

Also "Noorin TV" has its own YouTube channel (https://www.youtube.com/c/NoorinTv2/videos) and used to have a website (https://web.archive.org/web/20210818232841/https://noorintv.net/en).